packages/components/src/internal/components/search/utils.spec.ts

```diff
@@ -1260,6 +1301,25 @@ describe('isValidFilterField', () => {
         expect(isValidFilterFieldExcludeLookups(field, queryInfo, SampleTypeDataType.exprColumnsWithSubSelect)).toBe(
             false
         );
+        expect(isValidFilterFieldSampleFinder(field, queryInfo, SampleTypeDataType.exprColumnsWithSubSelect)).toBe(
+            true
+        );
+    });
+
+    test('mult-value lookup field', () => {
+        const field = QueryColumn.create({ name: 'test', lookup: { isPublic: true }, multiValue: true });
+        const queryInfo = QueryInfo.create({
+            schemaName: 'test',
+            name: 'query',
+            supportGroupConcatSubSelect: true,
```

Comment:
Can consider adding a test for supportGroupConcatSubSelect=false as well.

Reply:
done